Content.Shared/_DV/CosmicCult/Components/CosmicGlyphComponent.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
```

Comment:
Remove unused using directive
```suggestion
using Robust.Shared.Audio;
```